ui/src/shared/middleware/errors.js

```diff
@@ -35,36 +41,39 @@ const errorsMiddleware = store => next => action => {
         message ===
         `This organization is private. To gain access, you must be explicitly added by an administrator.` // eslint-disable-line quotes
       ) {
-        store.dispatch(notify(alertType, message))
+        store.dispatch(
+          publishNotification(multitenancyUserNotifications.unauthorized)
+        )
+      }
+
+      if (_.startsWith(message, 'Welcome to Chronograf')) {
```

Comment:
what is this supplementing that was lost or different before? seems fragile to check the content of the text